ui/src/app/views/workspace/component.spec.ts

```diff
@@ -149,26 +115,37 @@ describe('WorkspaceComponent', () => {
         { provide: ActivatedRoute, useValue: activatedRouteStub },
         { provide: ProfileService, useValue: new ProfileServiceStub() },
         { provide: UserService, useValue: new UserServiceStub() },
+        {
+          provide: CdrVersionStorageService,
+          useValue: new CdrVersionStorageServiceStub({
+            defaultCdrVersion: WorkspacesServiceStub.stubWorkspace().cdrVersionId,
+            items: [{
+              name: 'cdr1',
+              cdrVersionId: WorkspacesServiceStub.stubWorkspace().cdrVersionId,
+              dataAccessLevel: DataAccessLevel.Registered,
+              creationTime: 0
+            }]
+          })
+        },
         {
           provide: ServerConfigService,
           useValue: new ServerConfigServiceStub({
             gsuiteDomain: 'fake-research-aou.org'
           })
         }
       ]}).compileComponents().then(() => {
-        workspacePage = new WorkspacePage(TestBed);
+        fixture = TestBed.createComponent(WorkspaceComponent);
+        updateAndTick(fixture);
       });
-      tick();
   }));
 
   it('displays correct notebook information', fakeAsync(() => {
```

Comment:
Is this test old? Are we still displaying the notebooks on the workspace about page?

Reply:
It's very old. We are still loading notebooks here though so we can put them in the scroll component.

This test is pretty useless though, I'm just going to drop this test case and leave only the research purpose check as a basic render verification.